src/main/java/grumblebug/DialogBox.java

```diff
@@ -0,0 +1,47 @@
+package grumblebug;
+
+import javafx.geometry.Pos;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.scene.Node;
```

Comment:
Should this be moved up so that the ordering of import statements is consistent as per the [coding standard](https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements)?